### PR TITLE
master LPS 158579 | Add tests on update and delete structured content folders by assetLibraryId and erc

### PR DESCRIPTION
**Pull request by yanan**

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:headless-frontend-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-frontend-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
ci:test:relevant

**Comment by yanan** [comment]
ci:test:headless-frontend-functional:nocompile

**Comment by yanan** [comment]
Failed test are unrelated to this pr, failed tests are [LPS-158454](https://issues.liferay.com/browse/LPS-158454):

AdvancedObjectAPI#CanReturnAggregationTermsDetailsInOneToManyRelationship
AdvancedObjectAPI#CanReturnAggregationTermsDetailsInOneToManyRelationshipAfterObjectDeletion
AdvancedObjectAPI#CanReturnAggregationTermsDetailsInOneToManyUpdatedRelationship

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentFolderAPI.macro [hunk 1]
```diff
@@ -54,7 +54,7 @@ definition {
 			var api = "asset-libraries/${assetLibraryId}/structured-content-folders";
 		}
 		else {
-			var api = "sites/${siteId}/structured-contents";
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentFolderAPI.macro:57]: is this endpoint not existing anymore? 🤔

Comment (yanan) [reply portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentFolderAPI.macro:57]: This is a typo, it should be with -folders

Comment (yanan) [reply portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentFolderAPI.macro:57]: This is a typo, it should be with -folders

**Review by magjed4289 — APPROVED** [review]

**Comment by yanan** [comment]
ci:forward

**Pull request closed without merging**